resources/builder.inc.sh

```diff
@@ -290,7 +290,7 @@ _builder_execute_child() {
     result=$?
     echo "${COLOR_RED}## $scope$action$target failed with exit code $result${COLOR_RESET}"
     exit $result
-  )
+  ) || exit $? # Required due to above subshell masking exit
```

Comment:
So... because line 292 is within the parentheses, its exit is masked and requires the newly-added code on line 293 seen here?

Reply:
Found this link that seems to suggest "yes": https://tldp.org/LDP/abs/html/subshells.html

Reply:
Yes. It's horrific.

Reply:
Yep